### ISSUE TITLE
关于代码复现结果的疑问

### ISSUE THREAD

**Issue by wjs011** [iSEE-Laboratory/VersReID#13]
您好，我在复现过程中，我严格按照README中的说明进行配置，cuda版本用的是11.3，所使用的预训练模型与数据集均来自您提供的链接，为什么部分数据集上的复现结果低于论文表格中报告的性能？不知是否是我在实验设置或理解上有所偏差，或是遗漏了某些关键步骤？冒昧向您请教，期待您的指点和回复，非常感谢！
我的配置：
2025-11-17 20:41:47,612 VersReID INFO: Saving model in the path :logs/ReID-Bank/
2025-11-17 20:41:47,612 VersReID INFO: Namespace(config_file='configs/ReID-Bank.yml', local_rank=0, opts=['MODEL.DEVICE_ID', "('0')", 'MODEL.PRETRAIN_PATH', 'ckpts/dino_mpda_vit_b_lupb.pth', 'OUTPUT_DIR', 'logs/ReID-Bank/\r'])
2025-11-17 20:41:47,613 VersReID INFO: Loaded configuration file configs/ReID-Bank.yml
2025-11-17 20:41:47,613 VersReID INFO: 
MODEL:
  ATT_DROP_RATE: 0.0
  BLOCK_WARMUP_EPOCHS: -1
  COS_LAYER: False
  DEVICE: 'cuda'
  DEVICE_ID: ('1')
  DEVIDE_LENGTH: 4
  DIST_TRAIN: False
  DROP_OUT: 0.0
  DROP_PATH: 0.2
  EMA: False
  EMA_M: 0.999
  EMA_S: 1
  ID_LOSS_TYPE: 'softmax'
  ID_LOSS_WEIGHT: 1.0
  IF_LABELSMOOTH: 'off'
  IF_WITH_CENTER: 'no'
  JPM: False
  LAST_STRIDE: 1
  METRIC_LOSS_TYPE: 'triplet'
  MULTI_SCENES: True
  NAME: 'transformer'
  NECK: 'bnneck'
  NO_MARGIN: True
  PRETRAIN_CHOICE: 'LUP'
  PRETRAIN_PATH: 'Input in command'
  RE_ARRANGE: True
  SHIFT_NUM: 5
  SHUFFLE_GROUP: 2
  SIE_CAMERA: False
  SIE_COE: 3.0
  SIE_VIEW: False
  STRIDE_SIZE: [16, 16]
  STU: False
  SCENE_PROMPT_TYPE: 1
  SCENE_PROMPT_NUMS: 2
  TRANSFORMER_TYPE: 'vit_base_patch16_224_TransReID'
  TRIPLET_LOSS_WEIGHT: 1.0
  TYPE: 'TR'

INPUT:
  SIZE_TRAIN: [384, 128]
  SIZE_TEST: [384, 128]
  PROB: 0.5 # random horizontal flip
  RE_PROB: 0.5 # random erasing
  PADDING: 10
  PIXEL_MEAN: [0.5, 0.5, 0.5]
  PIXEL_STD: [0.5, 0.5, 0.5]

DATASETS:
  NAMES: ('combine')
  COMBINE_NAMES: [ 'dslr_cuhk03', 'market1501', 'msmt17', 'celebrity', 'prcc', 'occluded_duke', 'sysumm01' ]
  COMBINE_TYPE: [0, 1, 1, 2, 2, 3, 4]
  COMBINE_PID: True
  ROOTS: ['/data0/VersReID/data/MLR-CUHK03/', '/data0/VersReID/data/Market-1501/','/data0/VersReID/data/MSMT17_V2/', '/data0/VersReID/data/Celeb-reID/', '/data0/VersReID/data/prcc', '/data0/VersReID/data/Occluded-DukeMTMC-Dataset/Occluded_Duke', '/data0/VersReID/data/SYSU-MM01/']

DATALOADER:
  COMBINE_SAMPLER: 'softmax_triplet'
  SAMPLER: 'softmax_triplet'
  NUM_INSTANCE: 4
  NUM_WORKERS: 8

SOLVER:
  BASE_LR: 0.0004
  BIAS_LR_FACTOR: 2
  CENTER_LOSS_WEIGHT: 0.0005
  CENTER_LR: 0.5
  CHECKPOINT_PERIOD: 120
  COSINE_MARGIN: 0.5
  COSINE_SCALE: 30
  END_EPOCHS: -1
  EVAL_PERIOD: 30
  GAMMA: 0.1
  IMS_PER_BATCH: 128
  LARGE_FC_LR: False
  LOG_PERIOD: 200
  MARGIN: 0.3
  MAX_EPOCHS: 120
  MIN_LR: 0.0
  MOMENTUM: 0.9
  OPTIMIZER_NAME: 'SGD'
  SEED: 1234
  STEPS: (40, 70)
  WARMUP_EPOCHS: 20
  WARMUP_FACTOR: 0.01
  WARMUP_METHOD: 'cosine'
  WEIGHT_DECAY: 0.0001
  WEIGHT_DECAY_BIAS: 0.0001
我在ReID-Bank分支的实验结果如下：
<img width="1285" height="1347" alt="Image" src="https://github.com/user-attachments/assets/dd54811e-da42-4129-acfe-ec859ee65867" />

**Comment by yanjk3** [comment]
看了下结果，好像还比较正常，我有两点建议你可以试一下。
- 尝试使用cuda 10.2，有人报告说cuda 12上跑出来性能偏低：[链接](https://github.com/WangTaoAs/PFD_Net/issues/3#issuecomment-1002036261)
- PRCC和mm01这俩数据集本身测试协议是多次测试取平均，这里存在一定波动的可能性，可以试下增大测试次数之类的算下均值和方差再确认下波动是否正常

**Comment by wjs011** [comment]
最近测试发现确实是cuda版本的问题，感谢解答！
